src/mui/field/ReferenceArrayField.js

```diff
@@ -90,7 +90,7 @@ ReferenceArrayField.propTypes = {
 
 const mapStateToProps = (state, props) => {
     const { record, source, reference } = props;
-    const ids = get(record, source);
+    const ids = get(record, source) || [];
```

Comment:
Thanks. But I am not sure how this can cause rerender? I tried to console log in render method and the result is the same.